application/src/main/java/org/togetherjava/tjbot/config/StarboardConfig.java

```diff
@@ -0,0 +1,44 @@
+package org.togetherjava.tjbot.config;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonRootName;
+
+import java.util.List;
+import java.util.Objects;
+
+@JsonRootName("starboard")
+public final class StarboardConfig {
+    private final List<String> emojiNames;
+    private final String channelName;
+
+    @JsonCreator(mode = JsonCreator.Mode.PROPERTIES)
+    public StarboardConfig(
+            @JsonProperty(value = "emojiNames", required = true) List<String> emojiNames,
+            @JsonProperty(value = "channelName", required = true) String channelName) {
+        this.emojiNames = Objects.requireNonNull(emojiNames);
+        this.channelName = Objects.requireNonNull(channelName);
+    }
+
+    /**
+     * Gets the list of emotes that are recognized by the starboard feature. A message that is
+     * reacted on with an emote in this list will be reposted in a special channel.
```

Comment:
"in a special channel" -> make the link to the other method, so that people understand the connection